telegram/files/chatphoto.py

```diff
@@ -24,25 +24,47 @@ class ChatPhoto(TelegramObject):
     """This object represents a chat photo.
 
     Attributes:
-        small_file_id (:obj:`str`): File identifier of small (160x160) chat photo.
-        big_file_id (:obj:`str`): File identifier of big (640x640) chat photo.
-
+        small_file_id (:obj:`str`): Unique file identifier of small (160x160) chat photo.
```

Comment:
Please use the same description as in the [API docs](https://core.telegram.org/bots/api#chatphoto)

Reply:
I'll do it later.